longest-substring-without-repeating/longest-substring-without-repeating.js

```diff
@@ -0,0 +1,25 @@
+// https://leetcode.com/problems/longest-substring-without-repeating-characters/
+
+function lengthOfLongestSubstring(s) {
+    if (s.length === 0) return 0;
+    if (s.length === 1) return 1;
+    const mySet = new Set();
+    let counter = 0;
+    let maxCounter = 0;
+    let left = 0;
+    for (let i = 0; i < s.length; i++) {
```

Comment:
zamknij to w jednej petli, 2 petle to juz za duza zlozonosc obliczeniowa a da sie to ogarnac jednym forem